integration-tests/integration/features/case-summary-hearing-outcome.feature

```diff
@@ -0,0 +1,98 @@
+Feature: Case progress
+  I need to add and see the hearing outcome for eligible cases
+  So that I can make important information, or actions, visible to others
+  As an authenticated user
+  I want to add, edit and see a hearing outcomes
+
+  Scenario: Display hearing outcome
+    Given I am an authenticated user
+    And I click the "Accept analytics cookies" button
+    Then I should not see the cookie banner
+
+    When I navigate to the "/B14LO/hearing/5b9c8c1d-e552-494e-bc90-d475740c64d8/defendant/8597a10b-d330-43e5-80c3-27ce3b46979f/summary" base route
```

Comment:
should these be hardcoded?

Reply:
Copy pasta, there are a lot of tests using this common step. Refactoring now deserves a ticket in itself.